examples/ecmascript5/ecmascript5_tokens.ts

```diff
@@ -258,16 +217,10 @@ module chevrotain.examples.ecma5 {
         static PATTERN = /'([^\\']+|\\([bfnrtv'"\\]|[0-3]?[0-7]{1,2}|x[0-9a-fA-F]{2}|u[0-9a-fA-F]{4}))*'/
     }
 
-    export class AbsRegularExpressionLiteral extends AbsLiteral { static PATTERN = lex.NA }
-    // TODO-SS: is this EmptyRegExp relevant for parser/tooling? maybe it is only relevant inside the lexer
-    //          as a special rule how to identify "emptyRegExp"
-    export class EmptyRegularExpression extends AbsRegularExpressionLiteral { static PATTERN = lex.NA }// /(?:)/
```

Comment:
yes.

An EmptyRegExp is still a valid RegExp.
'(?:)' is just a "none capturing group".
http://stackoverflow.com/questions/3512471/non-capturing-group